src/test/java/com/google/sps/ListOrgServletTest.java

```diff
@@ -0,0 +1,179 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+ 
+package com.google.sps;
+ 
+import static org.mockito.Mockito.*;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.QueryResultList;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.GivrUser;
+import com.google.sps.servlets.ListOrganizationsServlet;
+import javax.servlet.http.HttpServletRequest;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.ArrayList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+ 
+/** */
+@RunWith(JUnit4.class)
+public final class ListOrgServletTest {
+ 
+  private final LocalServiceTestHelper helper = new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+ 
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+ 
+  /* The masterEntityList is a list of entities that are populated into the local datastore, and also exist 
+   * in an arraylist. For each test, the datastore applies the Query received from the servlet, and then
+   * compares that query to a hardcoded list of expected entities given that tests parameters / user fields */
+  private ArrayList<Entity> masterEntityList = new ArrayList<>();
+ 
+  @Before
+  public void setUp() {
+    helper.setUp();
+ 
```

Comment:
A top level comment as an ascii table would be helpful to understand the test below.